### DISCUSSION TITLE
When trying to export as .glb, Model Editor throws a "Uncaught (in promise) Error: Sorry, non-ASCII characters are not yet supported." error

### DISCUSSION THREAD

**Discussion by jackyliang** [google/model-viewer#2228] [Q&A]
### Description

I imported a .glb file to do some quick metal roughness edits in [Model Editor](https://modelviewer.dev/editor/), but when I try to export the edits, in my console, I get the following errors. And no exports happen. 

![image](https://user-images.githubusercontent.com/4315746/95553664-d7479f00-09c3-11eb-9e81-fd830d1e1f3c.png)

Here is the .gltf/.glb dragged into a validator. I get no errors. 

```
{
    "uri": "scene.gltf",
    "mimeType": "model/gltf+json",
    "validatorVersion": "2.0.0-dev.3.2",
    "validatedAt": "2020-10-09T22:31:49.597Z",
    "issues": {
        "numErrors": 0,
        "numWarnings": 0,
        "numInfos": 0,
        "numHints": 0,
        "messages": [],
        "truncated": false
    },
    "info": {
        "version": "2.0",
        "generator": "Sketchfab-9.46.0",
        "resources": [
            {
                "pointer": "/buffers/0",
                "mimeType": "application/gltf-buffer",
                "storage": "external",
                "uri": "scene.bin",
                "byteLength": 679896
            }
        ],
        "animationCount": 0,
        "materialCount": 1,
        "hasMorphTargets": false,
        "hasSkins": false,
        "hasTextures": false,
        "hasDefaultScene": true,
        "drawCallCount": 1,
        "totalVertexCount": 14229,
        "totalTriangleCount": 28200,
        "maxUVs": 0,
        "maxInfluences": 0,
        "maxAttributes": 2
    }
}
```

#### Live Demo

Download this .glb file, drag it into Model Editor, and try to immediately export it to .glb: https://www.dropbox.com/s/boww9uemd87oywo/out.glb?dl=0

### Version

N/A

### Browser Affected

- [x] Chrome
- [x] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [x] MacOS
- [x] Windows

**Comment by jackyliang** [comment]
Are there any possible quick fixes or work arounds? I have a demo I am trying to work on and it would be nice to make some textural changes, because my model right now is just a simplistic looking gray box

**Comment by elalish** [comment]
Not sure about quick fixes; if you convert it to glTF with `gltf-pipeline` (on npm), then you could probably delete any non-ASCII characters with a text editor and try again. Or open in Blender and re-export? It's definitely a bug on our side, but it's not going to be fixed immediately.

**Comment by jackyliang** [comment]
@elalish thanks Ela! I am quite new to this data structure and file format. Do you know which characters I should be looking for within the .glTF file structure in order to find the offending text? Like, is there a specific key I should look for?

**Comment by elalish** [comment]
Anything that's not roman characters. There's probably a text editor function that does to ASCII conversion automatically, but you'd have to search around.

**Comment by jackyliang** [comment]
Thank you! I fixed it. Turns out it was a pesky em-dash in the file.
